mozregression/utils.py

```diff
@@ -60,6 +61,14 @@ def names(self):
 CACHE_SESSION = None
 
 
+def retry_get(args, **kwargs):
```

Comment:
Hm I may have been unclear here. It's all good except that I would like the function to be like this:
def retry_get(url, **kwargs):
    return redo.retry(..., args=(url,), kwargs=kwargs)
So a caller will just issue:
retry_get(self.pushlog_url()) instead of retry_get(args=(self.pushlog_url(),))
and
retry_get(url, auth=auth) instead of retry_get(args=(url,), kwargs=dict(auth=auth))